### ISSUE TITLE
locations list and create new location at /admin is broken

### ISSUE THREAD

**Issue by XiaNi** [xr3ngine/xr3ngine#751]
https://localhost:3000/admin
does not show selects values (scene, type)
both throw errors in console and stay empty.
![image](https://user-images.githubusercontent.com/273573/100088477-d88c3880-2e58-11eb-86bf-0160e9e07091.png)

```
react_devtools_backend.js:2430
Material-UI: You have provided an out-of-range value `private` for the select component.
Consider providing a value that matches one of the available options or ''.
The available values are "". 
    at SelectInput (https://localhost:3000/_next/static/chunks/pages/admin.js?ts=1606215861673:29794:24)
    at div
    at InputBase (https://localhost:3000/_next/static/chunks/pages/admin.js?ts=1606215861673:22185:30)
    at WithStyles (https://localhost:3000/_next/static/chunks/pages/admin.js?ts=1606215861673:185724:31)
    at Input (https://localhost:3000/_next/static/chunks/pages/admin.js?ts=1606215861673:21590:32)
    at WithStyles (https://localhost:3000/_next/static/chunks/pages/admin.js?ts=1606215861673:185724:31)
    at Select (https://localhost:3000/_next/static/chunks/pages/admin.js?ts=1606215861673:29495:32)
    at WithStyles (https://localhost:3000/_next/static/chunks/pages/admin.js?ts=1606215861673:185724:31)
    at div
    at FormControl (https://localhost:3000/_next/static/chunks/pages/admin.js?ts=1606215861673:18235:24)
    at WithStyles (https://localhost:3000/_next/static/chunks/pages/admin.js?ts=1606215861673:185724:31)
    at div
    at Transition (https://localhost:3000/_next/static/chunks/pages/admin.js?ts=1606215861673:262968:30)
    at Fade (https://localhost:3000/_next/static/chunks/pages/admin.js?ts=1606215861673:17580:24)
    at Unstable_TrapFocus (https://localhost:3000/_next/static/chunks/pages/admin.js?ts=1606215861673:39724:24)
    at div
    at Portal (https://localhost:3000/_next/static/chunks/pages/admin.js?ts=1606215861673:28631:24)
    at Modal (https://localhost:3000/_next/static/chunks/pages/admin.js?ts=1606215861673:25783:83)
    at div
    at LocationModal (https://localhost:3000/_next/static/chunks/pages/admin.js?ts=1606215861673:278482:20)
    at ConnectFunction (https://localhost:3000/_next/static/chunks/pages/_app.js?ts=1606215861673:102504:75)
    at div
    at Paper (https://localhost:3000/_next/static/chunks/pages/admin.js?ts=1606215861673:27545:23)
    at WithStyles (https://localhost:3000/_next/static/chunks/pages/admin.js?ts=1606215861673:185724:31)
    at AdminConsole (https://localhost:3000/_next/static/chunks/pages/admin.js?ts=1606215861673:278999:26)
    at ConnectFunction (https://localhost:3000/_next/static/chunks/pages/_app.js?ts=1606215861673:102504:75)
    at section
    at Layout (https://localhost:3000/_next/static/chunks/pages/admin.js?ts=1606215861673:287587:76)
    at ConnectFunction (https://localhost:3000/_next/static/chunks/pages/_app.js?ts=1606215861673:102504:75)
    at ThemeProvider (https://localhost:3000/_next/static/chunks/pages/admin.js?ts=1606215861673:184428:24)
    at AdminConsolePage
    at ConnectFunction (https://localhost:3000/_next/static/chunks/pages/_app.js?ts=1606215861673:102504:75)
    at Provider (https://localhost:3000/_next/static/chunks/pages/_app.js?ts=1606215861673:102217:20)
    at Be (https://localhost:3000/_next/static/chunks/pages/_app.js?ts=1606215861673:108692:67)
    at MyApp (https://localhost:3000/_next/static/chunks/pages/_app.js?ts=1606215861673:114922:25)
    at ConnectFunction (https://localhost:3000/_next/static/chunks/pages/_app.js?ts=1606215861673:102504:75)
    at withRedux(Connect(MyApp)) (https://localhost:3000/_next/static/chunks/pages/_app.js?ts=1606215861673:87732:28)
    at ErrorBoundary (https://localhost:3000/_next/static/chunks/main.js?ts=1606215861673:18763:43)
    at ReactDevOverlay (https://localhost:3000/_next/static/chunks/main.js?ts=1606215861673:18929:21)
    at Container (https://localhost:3000/_next/static/chunks/main.js?ts=1606215861673:11619:5)
    at AppContainer (https://localhost:3000/_next/static/chunks/main.js?ts=1606215861673:12087:24)
    at Root (https://localhost:3000/_next/static/chunks/main.js?ts=1606215861673:12203:25)
```

**Comment by XiaNi** [comment]
@shawticus @barankyle 
this issue is somehow related to commit https://github.com/xr3ngine/xr3ngine/commit/bfe9f8d379cc70d3cc7a200f45c2fe7236ffa2b5

**Comment by barankyle** [comment]
Fixed by #755
